scripts/compare_testsuite_log.py

```diff
@@ -369,17 +371,20 @@ def is_result_valid(log_path: str):
                     return False
         return True
 
+def run(previous_hash: str, previous_log: str, current_hash: str, current_log: str, output_markdown: str):
```

Comment:
Can we update this to a more descriptive name? When I was reading the other file I got a bit confused as to what `run` did.